packages/itwin/reports-config-widget/package.json

```diff
@@ -99,7 +99,7 @@
     "@typescript-eslint/eslint-plugin": "^5.3.1",
     "@typescript-eslint/parser": "^5.10.1",
     "chai": "^4.2.0",
-    "cpx2": "^3.0.0",
+    "cpx2": "^4.0.0",
```

Comment:
Might have been an accidental version bump here.

Reply:
Lockfile has to reflect the change as well.

Reply:
I updated the lockfile.